tests/managed_services/mas_debezium/conftest.py

```diff
@@ -0,0 +1,205 @@
+import logging
+
+import pytest
+import rhoas_kafka_instance_sdk
+from constants import KAFKA_NAME, TEST_RECORD, TEST_TOPIC
+from consumer_pod import ConsumerPod
+from ocp_resources.namespace import Namespace
+from ocp_resources.pod import Pod
+from ocp_resources.utils import TimeoutExpiredError, TimeoutSampler
+from ocp_utilities.infra import cluster_resource
+from rhoas_kafka_instance_sdk.api import acls_api, records_api, topics_api
+from rhoas_kafka_instance_sdk.model.acl_binding import AclBinding
+from rhoas_kafka_instance_sdk.model.acl_operation import AclOperation
+from rhoas_kafka_instance_sdk.model.acl_pattern_type import AclPatternType
+from rhoas_kafka_instance_sdk.model.acl_permission_type import AclPermissionType
+from rhoas_kafka_instance_sdk.model.acl_resource_type import AclResourceType
+from rhoas_kafka_instance_sdk.model.config_entry import ConfigEntry
+from rhoas_kafka_instance_sdk.model.new_topic_input import NewTopicInput
+from rhoas_kafka_instance_sdk.model.record import Record
+from rhoas_kafka_instance_sdk.model.topic_settings import TopicSettings
+from rhoas_kafka_mgmt_sdk.model.kafka_request_payload import KafkaRequestPayload
+from rhoas_service_accounts_mgmt_sdk.model.service_account_create_request_data import (
+    ServiceAccountCreateRequestData,
+)
+
+
+LOGGER = logging.getLogger(__name__)
+WAIT_STATUS_TIMEOUT = 120
+
+
+@pytest.fixture(scope="class")
+def kafka_instance(kafka_mgmt_api_instance):
+    _async = True
+    kafka_request_payload = KafkaRequestPayload(
+        cloud_provider="aws",
+        name=KAFKA_NAME,
+        region="us-east-1",
+        plan="standard.x1",
+        reauthentication_enabled=True,
+    )
+    kafka_create_api = kafka_mgmt_api_instance.create_kafka(
+        _async=_async, kafka_request_payload=kafka_request_payload
+    )
+    assert (
+        kafka_create_api.status == "accepted"
+    ), f"Failed to create a kafka instance. API response:\n{kafka_create_api}"
+
+    yield kafka_create_api
+
+    kafka_mgmt_api_instance.delete_kafka_by_id(
+        async_req=True, _async=_async, id=kafka_create_api.id
+    )
+
+
+@pytest.fixture(scope="class")
+def kafka_instance_ready(kafka_mgmt_api_instance, kafka_instance):
+    try:
```

Comment:
here the log - "wait for kafka instance to be ready" or something similar

Reply:
done